recipes/gnutls/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gnutls" %}
+{% set version = "3.5.15" %}
+{% set sha256 = "046081108b8b1fe455a13a4c5a4eaa0368e185b678f1670fe09a11a2d7ecfad5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: ftp://ftp.gnutls.org/gcrypt/{{ name }}/v3.5/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - pkgconfig.patch
+
+
+build:
+  number: 0
+  skip: True  # [win]  
+
+requirements:
+  build:
+    - gmp 6.1.*
+    - nettle
+    - zlib 1.2.8 
+    - toolchain3
+    - libidn11
+    - pkg-config  # [osx]
+    - openssl 1.0.*
+
+  run:
+    - gmp 6.1.*
+    - nettle
+    - zlib 1.2.8  # [osx]
+    - libidn11  # [osx]
```

Comment:
Why is this `osx` only?

Reply:
I made this osx specific because when building in circle CI, those two libraries are included directly in the system so I thought they were not needed.

Reply:
For `zlib` we prefer using the conda package and for `libidn11` as well.